ops/dataloader/src/main/java/gov/doj/Driver.java

```diff
@@ -8,7 +8,11 @@
 
 public class Driver {
```

Comment:
At the moment, this class still needs refactoring to include executing loading logic for AODATEX database.